### PR TITLE
misc quality of life improvements

### PR DESCRIPTION
**Pull request by gdementen**

### CODE REVIEW

File: doc/source/changes/version_0_22.rst.inc [hunk 1]
```diff
@@ -151,42 +159,48 @@ Miscellaneous improvements
...
+  var, ...). Closes :issue:`41`.
 
-    >>> anonymous = Axis('0..100')
-    >>> age = Axis('age=0..100')
-    >>> gender = Axis('M,F', 'gender')
+* improved documentation of plot method (closes :issue:`169`).
+
+* improved auto-completion in interactive consoles for both LArray and Session objects. LArray objects can now
+  complete keys within [].
+
```
Comment (gdementen) [inline doc/source/changes/version_0_22.rst.inc]: yes! It is a relatively new feature in ipython. I should probably specify that it's ipython specific.

**Comment by gdementen** [comment]
I did not test sessions with numpy arrays. Thinking of which, the pickle part needs tests. As for #217, I do not see what is the relationship with this PR?!

File: doc/source/changes/version_0_22.rst.inc (hunk 1 above)
Comment (alixdamman) [inline doc/source/changes/version_0_22.rst.inc]: My question was not about `tab` but about to specify `using tab` after `within []`.

**Comment by alixdamman** [comment]
Well, for me #217 is just a small misc. improvement. It does not need a specific PR.

File: doc/source/changes/version_0_22.rst.inc (hunk 1 above)
Comment (gdementen) [inline doc/source/changes/version_0_22.rst.inc]: hehe, ok :)

**Comment by alixdamman** [comment]
In Session.py - class FileHandler - method read_arrays(), res shouldn't be declared as an OrderedDict instead of a simple dict (line 87)?

**Comment by gdementen** [comment]
It should indeed.

**Pull request merged**